### ISSUE TITLE
moveit planners plugin?

### ISSUE THREAD

**Issue by shengyangzhuang** [lbr-stack/lbr_fri_ros2_stack#195]
Dear mhubii,

I am trying to configure curobo moveit plugin with lbr-stack. I faced some similar issue that I previously wanted to ask. Currently, the moveit cannot choose any planner and always uses default RRTConnect. Could you provide some guidance on how to configure other planners? Thank you for any help you can provide.

![Screenshot from 2024-07-08 14-32-33](https://github.com/lbr-stack/lbr_fri_ros2_stack/assets/100099974/45b698a4-9f1a-41f6-b977-b863b165a998)

Warm regards,

S.Z.

**Comment by mhubii** [comment]
good point, this configuration is probably missing right now.

Putting for reference https://nvidia-isaac-ros.github.io/repositories_and_packages/isaac_ros_cumotion/isaac_ros_cumotion_moveit/index.html

**Comment by shengyangzhuang** [comment]
Thanks for your response. I tried it yesterday as well. It was all about docker development. I am not quite familiar with docker. I tried to mount the lbr-stack inside the container. Unfortunately it could not be built. Then I tried to install it inside the container, it could not be built either. [Here](https://nvidia-isaac-ros.github.io/concepts/docker_devenv/index.html#development-environment) is the instruction I followed.

I was trying to add my own layer including lbr-stack on top of the base images provided in Isaac ROS. I couldn't quite understand their instructions. I would be appreciated if you could provide me with some insights.

This is my directory structure
```
~/.isaac_ros_common-config
/home/szhuang/workspaces/isaac_ros-dev
     /home/szhuang/workspaces/isaac_ros-dev/ros2_ws
         /home/szhuang/workspaces/isaac_ros-dev/ros2_ws/lbr-stack_docker
         /home/szhuang/workspaces/isaac_ros-dev/ros2_ws/lbr-stack_docker/Dockerfile.lbr
         /home/szhuang/workspaces/isaac_ros-dev/ros2_ws/lbr-stack_docker/container_build.sh
```

I then have ~/.isaac_ros_common-config as follows:
```
CONFIG_IMAGE_KEY= Dockerfile.lbr
CONFIG_DOCKER_SEARCH_DIRS=(/home/szhuang/workspaces/isaac_ros-dev/ros2_ws/lbr-stack_docker)
```
Then I run 
```
~workspaces/isaac_ros-dev/src/isaac_ros_commom/scripts/run_dev.sh -d /home/szhuang/ros2_ws_mounted
```

It told me it couldn't find the Dockerfile.lbr, even though I specified the directory already. Besides, I used the dockerfile provided by lbr-stack directly. I might need to make some changes as well, right?

I am trying to install [it](https://nvidia-isaac-ros.github.io/getting_started/isaac_apt_repository.html) natively today and will see if it can work without docker.

**Comment by shengyangzhuang** [comment]
I think I finally sort it out :)
I can try configuring all the planners with lbr-stack, including cumotion, chomp, and others inside ompl

**Comment by mhubii** [comment]
did you modify this line?

https://github.com/lbr-stack/lbr_fri_ros2_stack/blob/519f5203518d8111ec9a98856ba711ce9bad99d0/lbr_bringup/launch_mixins/lbr_bringup.py#L70

**Comment by shengyangzhuang** [comment]
yes, I think I will need to modify that line. I'm working on the [XRDF ](https://nvidia-isaac-ros.github.io/repositories_and_packages/isaac_ros_cumotion/isaac_ros_cumotion_moveit/index.html#quickstart)file mentioned here. Then I will modify that to see if it works.

**Comment by shengyangzhuang** [comment]
Update:

So I just setup cumotion with iiwa7. Somehow it just got stuck in planning stage even though it was a simple movement.

There isn't any error and everything looks fine. I would be grateful if anyone with relevant experience could offer me some insights. I posted the same issue on cumotion github repo.

> https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_cumotion/issues/8
